test/bdd/pkg/context/context.go

```diff
@@ -100,6 +101,7 @@ func NewBDDContext(caCertPaths ...string) (*BDDContext, error) {
 type BDDInteropContext struct {
 	TrustBlocEDVHostURL        string
 	TrustBlocEDVClient         *edvclient.Client
+	TrustBlocEDVDataVaultID    string
```

Comment:
Hmm... how come we store the vault ID for the TrustBloc one but not for the Transmute one?

Reply:
I can store the Transmute one too but I don't see it being used anywhere for now.

Reply:
So before your change, the way I was comparing between the TrustBloc and Transmute vault locations was a little different since they were randomly generating their vault IDs and we weren't... I'm not exactly sure why I didn't try comparing them in a more 1:1 way before, but now that they both work identically, you should be able to have this precise symmetry between the "TrustBloc" and "Transmute" variables/checks... Let me know if that doesn't quite make sense and I'll clarify.